Package/ExternalCliAction.cs

```diff
@@ -0,0 +1,136 @@
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using OpenTap.Cli;
+
+namespace OpenTap.Package
```

Comment:
Split this file into one for each class.

No need to add the internal specifier for each class, that is implied.